po/de.po

```diff
@@ -3291,32 +3910,37 @@ msgstr "Interner Fehler"
 msgid "Invalid address $0"
 msgstr "Ungültiger Schlüssel"
 
-#: pkg/systemd/host.js:1452 pkg/systemd/shutdown.js:143
+#: pkg/subscriptions/subscriptions-client.js:195
+#, fuzzy
```

Comment:
Why did you add the fuzzy markers to a lot of your new translations? They won't be used like this, just suggested as a starting point in translation UIs.